thecampuscookbook/thecampuscookbook/models.py

```diff
@@ -0,0 +1,54 @@
+from django.utils.timezone import timezone
+from tkinter import CASCADE
+from django.db import models
+
+
+class USERS(models.Model):
+    user_id = models.AutoField(primary_key=True)
+    avatar_image = models.ImageField(upload_to='/media/',blank=True,null=True)
+    name = models.CharField(max_length=200)
+    user_name = models.CharField(max_length=200,unique = True)
+    email = models.EmailField(max_length=150,unique=True)
+    password = models.CharField(max_length=128,)
+    join_date = models.DateTimeField(default = timezone.now)
+    def __str__(self):
+        return self.user_name
+
+class RECIPES(models.Model):
+    recipe_id = models.AutoField(primary_key=True)
+    user_id = models.ForeignKey(USERS,on_delete=models.CASCADE)
+    title = models.CharField(max_length=100)
+    origin = models.CharField(max_length=100)
+    category = models.CharField(max_length=100)
```

Comment:
@Alexa3636 @ShreyasNerd009 shouldn't we also have a `Category` entity?
We can use it as a foreign key in the Recipe. Having a `Category` entity also ensures that a Recipe cannot be added to a non-existent category.